plugins/node/opentelemetry-instrumentation-mongodb/src/instrumentation.ts

```diff
@@ -36,23 +36,47 @@ import {
 import { MongoDBInstrumentationConfig, CommandResult } from './types';
 import {
   CursorState,
+  ServerSession,
   MongodbCommandType,
   MongoInternalCommand,
   MongoInternalTopology,
   WireProtocolInternal,
   V4Connection,
 } from './internal-types';
+import { V4Connect, V4Session } from './types';
 import { VERSION } from './version';
+import { UpDownCounter } from '@opentelemetry/api';
 
 /** mongodb instrumentation plugin for OpenTelemetry */
 export class MongoDBInstrumentation extends InstrumentationBase {
+  private _connectionsUsage!: UpDownCounter;
+  private _poolName!: string;
+
   constructor(protected override _config: MongoDBInstrumentationConfig = {}) {
     super('@opentelemetry/instrumentation-mongodb', VERSION, _config);
   }
 
+  override _updateMetricInstruments() {
+    this._connectionsUsage = this.meter.createUpDownCounter(
+      'db.client.connections.usage',
+      {
+        description:
+          'The number of connections that are currently in the state referenced by the attribute "state".',
+        unit: '{connections}',
```

Comment:
Not sure if the one in the semconv is a typo or this one here (see https://github.com/open-telemetry/opentelemetry-specification/blob/c210cf5884cf2c06dc1d1a6a34a4ea76c542841f/specification/metrics/semantic_conventions/database-metrics.md?plain=1#L34) :slightly_smiling_face: 
```suggestion
        unit: '{connection}',
```

Reply:
I looked [here](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#instrument-units) and it says: "All non-units that use curly braces to annotate a quantity need to match the grammatical number of the quantity it represent. For example if measuring the number of individual requests to a process the unit would be {request}, not {requests}.".
If I understand correctly we should use {connection} and not {connections}. what do you think?

Reply:
Ah yes, that's also what I understand when reading the spec there. I agree, I think we should switch to `{connection}`. :slightly_smiling_face:

Reply:
I should change it in mysql too https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1220